swan-lake/development-tutorials/deployment-guide/serverless-deployment.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: ballerina-cloud-left-nav-pages-swanlake
+title: Serverless deployment
+description: Serverless architecture allows developers to focus on writing application logic without worrying about managing servers or infrastructure. AWS Lambda automatically handles provisioning, scaling, and maintaining resources, letting you concentrate solely on your code. Following are the  Ballerina functions in a serverless environment, 
+keywords: ballerina, programming language, services, cloud, kubernetes, docker
+active: ballerina-deployment-guideLines
+intro: Serverless architecture allows developers to focus on writing application logic without worrying about managing servers or infrastructure. AWS Lambda automatically handles provisioning, scaling, and maintaining resources, letting you concentrate solely on your code. Following are the  Ballerina functions in a serverless environment, 
+---
+
+1. [AWS Lambda](#aws-lambda)
+2. [Azure Functions](#azure-function)
+
+## AWS Lambda
+
+In this guide, we'll walk through deploying a serverless application using AWS Lambda with Ballerina. You'll learn how to set up, deploy, and manage your application, simplifying your production workflow and eliminating the need to manage servers.
+An AWS Lambda function can be triggered by various AWS services. You can find the list of supported notification types below.
+- Direct invocation
+- Simple Queue Service (SQS)
+- Simple Storage Service (S3)
+- DynamoDB 
+- Simple Email Service (SES)
+- API Gateway
+
+This example demonstrates how to write a simple echo function in AWS Lambda.
+
+For more information, see [the AWS Lambda learn guide](https://ballerina.io/learn/aws-lambda/).
```

Comment:
Now we don't have a learn section tile for this [aws-lambda ](https://ballerina.io/learn/aws-lambda/)page and [azure-functions](https://ballerina.io/learn/azure-functions/) page. How this will impact the left navigation when the reader lands on such a page? Better to include them under Serverless deployment in left nav. 

And also can we include next level titles in the landing page tile as well as indented links under serverless deployment. Adding @sm1990 as well.

<img width="405" alt="image" src="https://github.com/user-attachments/assets/11d9981e-222d-45cf-b013-3a7abf02c117">

Reply:
@anupama-pathirage & @lnash94 
> Now we don't have a learn section tile for this [aws-lambda ](https://ballerina.io/learn/aws-lambda/)page and [azure-functions](https://ballerina.io/learn/azure-functions/) page. How this will impact the left navigation when the reader lands on such a page? Better to include them under Serverless deployment in left nav.

We can create a new section called `Serverless deployment` under `Deployment guidelines`. 
Once you click on `Serverless deployment` in left nav, it will expand the section in the left nav and show 3 pages. 

- Serverless deployment overview
- AWS
- Azure


> And also can we include next level titles in the landing page tile as well as indented links under serverless deployment.

We are currently showing headings/titles only up to the 3rd level in the `Learn` landing page. If we are going deeper, we'll have to update other sections as well.